bundler/spec/native_helpers/functions/conflicting_dependency_resolver_spec.rb

```diff
@@ -36,6 +36,95 @@
       )
     end
 
+    context "for nested transitive dependencies" do
+      let(:gemfile_fixture_name) { "transitive_blocking" }
+      let(:lockfile_fixture_name) { "transitive_blocking.lock" }
+      let(:dependency_name) { "activesupport" }
+      let(:target_version) { "6.0.0" }
+
+      it "returns a list of dependencies that block the update" do
+        expect(conflicting_dependencies).to match_array(
+          [
+            {
+              "explanation" => "rails (5.2.0) requires activesupport (= 5.2.0) via rails (5.2.0)",
```

Comment:
Should this include `via rails`?

Reply:
Looks like there's some code that guards against this
```suggestion
              "explanation" => "rails (5.2.0) requires activesupport (= 5.2.0)",
```

Reply:
Oh right, let me double check if it's just the spec that's wrong or some of the logic is wonkey